blas/gonum/level2double.go

```diff
@@ -1343,10 +1345,10 @@ func (Implementation) Dtbsv(ul blas.Uplo, tA blas.Transpose, d blas.Diag, n, k i
 	}
 }
 
-// Dsbmv performs
+// Dsbmv performs the matrix-vector operation
 //  y = alpha * A * x + beta * y
-// where A is an n×n symmetric banded matrix, x and y are vectors, and alpha
-// and beta are scalars.
+// where A is an n×n symmetric band matrix with k super-diagonals, x and y are
```

Comment:
"k super-diagonals" or "k+1 diagonals"?

Reply:
`k` does not include the main diagonal and since the matrix is symmetric, the total bandwidth is `k + 1 + k`. This is consistent with `Dgbmv`. `k+1` diagonals is used in `Dtbmv` which also fits the same scheme. So "k super-diagonals" is correct.

Reply:
Thanks.